apps/docs/src/common/pages/get-started/design-assets.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const applyPrototype1 = require('../../../../assets/images/prototype-screen01.png').default;
+const applyPrototype2 = require('../../../../assets/images/prototype-screen02.png').default;
```

Comment:
These lines should be removed.

And maybe you want import the Mural PDF here instead?

Reply:
I've fixed this.